### PR TITLE
add more urls of whatsapp and instagram

### PR DESCRIPTION
**Pull request by xhdix**
just to see the results

### CODE REVIEW

File: ooni.json [hunk 1]
```diff
@@ -85,6 +85,38 @@
                 "https://scontent-cdg2-1.cdninstagram.com/favicon.ico",
                 "https://instagram.fdtm1-1.fna.fbcdn.net/favicon.ico",
                 "https://external.xx.fbcdn.net/robots.txt",
+                "https://instagram.fevn1-1.fna.fbcdn.net/favicon.ico",
+                "https://instagram.fevn1-4.fna.fbcdn.net/favicon.ico",
+                "https://instagram.fevn1-2.fna.fbcdn.net/favicon.ico",
+                "https://instagram.fevn1-3.fna.fbcdn.net/favicon.ico",
+                "https://scontent.fevn1-3.fna.fbcdn.net/favicon.ico",
+                "https://media0.giphy.com/robots.txt",
+                "https://media3.giphy.com/robots.txt",
```
Comment (xhdix) [inline ooni.json:94]:
```suggestion
                "https://media3.giphy.com/robots.txt"
```

**Comment by xhdix** [comment]
[test]

File: ooni.json [hunk 2]
```diff
@@ -85,6 +85,38 @@
                 "https://scontent-cdg2-1.cdninstagram.com/favicon.ico",
                 "https://instagram.fdtm1-1.fna.fbcdn.net/favicon.ico",
                 "https://external.xx.fbcdn.net/robots.txt",
+                "https://instagram.fevn1-1.fna.fbcdn.net/favicon.ico",
+                "https://instagram.fevn1-4.fna.fbcdn.net/favicon.ico",
+                "https://instagram.fevn1-2.fna.fbcdn.net/favicon.ico",
+                "https://instagram.fevn1-3.fna.fbcdn.net/favicon.ico",
+                "https://scontent.fevn1-3.fna.fbcdn.net/favicon.ico",
+                "https://media0.giphy.com/robots.txt",
+                "https://media3.giphy.com/robots.txt"
```
Comment (xhdix) [inline ooni.json]:
```suggestion
                "https://media3.giphy.com/robots.txt",
```

Comment (xhdix) [reply ooni.json]: now, it should be fine?

**Review by bassosimone — APPROVED** [review]
🐳

**Pull request merged**